### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Serene-Arc**
A while back, beets added the option to explicitly disable MusicBrainz, meaning the work around shown in the example configuration is no longer needed.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,70 +2,70 @@
 
 This is a plugin that allows Beets to manage audiobook collections.
 
-It fetches audiobook metadata via the Audible and [Audnex API](https://github.com/laxamentumtech/audnexus). With this data, it ensures books have the correct tags and makes the collection ready to be served by Plex, Audiobookshelf or Booksonic.
+It fetches audiobook metadata via the Audible and [Audnex API](https: // github.com / laxamentumtech / audnexus). With this data, it ensures books have the correct tags and makes the collection ready to be served by Plex, Audiobookshelf or Booksonic.
 
-## Motivation
+# Motivation
```
Comment (Neurrone) [inline readme.md:7]: Could you remove unrelated changes so that the PR would be easier to review?

**Review by Neurrone — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: readme.md (hunk 1 above)
Comment (Serene-Arc) [reply readme.md:7]: Oh, so sorry. Apparently the auto-format tool from another project is still being applied, I'll fix it up.

**Comment by Neurrone** [comment]
Thanks for removing those extra changes.

One last question, is this new option documented elsewhere? I don't see it anywhere in the Beets documentation.

**Comment by Serene-Arc** [comment]
Ah well this is embarrassing, I must apologise. Apparently this option is only in the latest version. The stable branch, the last release, is over a year old, and this option has been added since then. So this PR is a bit premature. Sorry about that.

If/when they do the next release, then this PR will be relevant, unless the user installs from the git repository. Using this option does remove a *lot* of errors from the output.

**Comment by Neurrone** [comment]
Np. I'll be happy to leave this open for now and merge it in once Beets has a new stable release.

**Review by Neurrone — APPROVED** [review]

**Review by Neurrone — APPROVED** [review]
Thanks @buswedg for verifying that this works in v1.6 stable. I've just tried it and it works for me as well.

Will go ahead and merge this PR.

Fixes #38
